source/cli/commands.md

```diff
@@ -1,194 +1,216 @@
 
-## `vijil login`
+To authenticate the CLI to your Vijil account, use:
 
-Used for authentication, required to use all other CLI commands. Use your login credentials for the GUI.
+`vijil login <username>`
 
+> This command requires a Vijil access token created from the GUI.
 
-### Arguments
+
+**Arguments**
 
 | Name | Possible values | Usage/Description
 |--|---|---|
-| `--username` | String | Registered Username of Vijil. |
-| `--token` | String | API Token of Vijil. |
+| `--token` | String | VIJIL access token from the GUI. |
+
+**Example**
+
+```console
+$ vijil login admin --token 'my-token'
+```
 
-### Example
+### Logout
+
+To log out from your Vijil account in the CLI, use:
+
+`vijil logout`
 
 ```console
-foo@bar:~$ vijil login --username my_username --token my_token
-Token verification successful. Configuration complete.
+$ vijil logout
 ```
 
+## Create
+
+### Token
+
+To create tokens for interacting with the model hubs, use:
+
+`vijil create token <model-hub>`
 
-## `vijil logout`
+You can choose from the following hubs:
+- `huggingface`
+- `octo`
+- `replicate`
+- `openai`
+- `together`
+- `anyscale`
+- `mistral`
 
-Logs out from CLI.
+**Arguments**
+| Name | Possible values | Usage/Description
+|--|---|---|
+| `--name` | String | Name of token. |
+| `--token` | String | Token from the model hub. |
+| `--is-primary` | [yes\|no] | Save token as default (yes or no). |
 
+**Example**
 ```console
-foo@bar:~$ vijil logout
+$ vijil create token octo --name 'token-name' --token 'your-token' --is-primary 'yes'
 ```
 
-## `vijil run`
+## Start
+
+### Evaluation
+
+Start an evaluation against a model using:
 
-Used to submit an evaluation.
+`vijil start evaluation`
 
-### Arguments
+**Arguments**
 | Name | Possible values | Usage/Description
 |--|---|---|
-| `--job-name`	| String	| Unique name for Job. |
-| `--model-type`	| String	| Model provider (e.g., huggingface, octo, replicate). |
+| `--model-hub`	| String	| Model hub (e.g., huggingface, octo, replicate). |
 | `--model-name`	| String	| Name of the model to be evaluated. |
 | `--dimension`	| String	| Trust dimensions (e.g., ethics, security). |
 | `--generations`	| Integer	| Number of generations for the evaluation. |
 | `--deployment-type` | String | Deployment type of model (e.g., private, public, local). |
-| `--token` | String | Token for Provoded model. |
+| `--token` | String | Token for accessing the provided model. |
 
-### Example
+**Example**
 
 ```console
-foo@bar:~$ vijil run --job-name jobname --model-type octo --model-name mistral-7b-instruct-fp16 --dimension ethics --deployment-type public --generations 1 --token 'your-token'
-Running evaluation for model type: octo, model name: mistral-7b-instruct-fp16
-Successfully Create Evaluation, Check Job Status by ID: 81f05220-27be-4a84-b707-354a71b7359e
+$ vijil start evaluation --model-hub 'octo' --model-name 'mistral-7b-instruct-fp16' --dimension 'ethics' --deployment-type 'public' --generations 1 --token 'your-token'
```

Comment:
I think you can leave out the quotes

Reply:
I added quotes for color-coding the difference between the key and the value.

<img width="848" alt="Screenshot 2024-02-21 at 2 41 22 PM" 
src="https://github.com/vijilAI/docs/assets/37476886/449a1907-2522-4e5c-9817-589b287ba993">

I can remove them if that's not necessary.

Reply:
aha, I see your point. I'm worried about users getting wrong ideas of how to use the commands, which ranks slightly higher than readability in my mind. one middle-ground would be to separate backslash+new line, eg
```console
vijil start evaluation --model-type octo \
   --model-name mistral-7b-instruct-fp16 \
   --dimension Security \
   --deployment-type public \
   --generations 1 \
   --token your-token
```

Reply:
Resolved https://github.com/vijilAI/docs/pull/15/commits/933ccea8330cd2542b744387736a15929a58813c